### ISSUE TITLE
ReactTestUtils has been moved

### ISSUE THREAD

**Issue by cranst0n** [japgolly/scalajs-react#359]
Looks like 15.5.x has moved ReactTestUtils

Excerpt from [React 15.5.0 blog post](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html)

> Currently, the React Test Utils live inside react-addons-test-utils.
> As of 15.5, we're deprecating that package and moving them to react-dom/test-utils instead:

Everything still works in my tests using `WebpackRequire`. I do however, get a few warnings on the console:

```
[error] Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.
```

**Comment by japgolly** [comment]
Yeah I decided to use the old reference for a while to give people a chance to migrate. There was already so much going on with the 1.0 release that I didn't want React < 15.5 breaking on people.

**Comment by japgolly** [comment]
Fixed in ac48a17b79400c5267cda355c2a60bb6dced4d38
From the next release `react-dom/test-utils` will be used.
